src/app.py

```diff
@@ -29,37 +29,21 @@ def say_hello_world():
     return {'result': "Flask says Hello World"}
 
 @app.route('/surgeo', methods=["GET", "POST"])
-def run_surgeo():
-    # TO-DO: @KAYLA
-    # Build out a public facing API that responds to a get request with the surname and zip code as query arguments. 
-    # The API should return the probabilities of each race prediction as a JSON object.
-    # Example of return object: {'AAPI': .56, 'Hispanic': .32, ..., 'White': .10}
+def internal_surgeo():
+    # API for internal use and testing only; will be deprecated in the future
+    # Required fields: 'surname', 'zipcode'
+
     surname = request.args.get('surname')
     zipcode = request.args.get('zipcode')
-    surname_series = pd.Series([surname])
-    zip_series = pd.Series([zipcode])
-
-    sg = surgeo.SurgeoModel()
-    sg_results = sg.get_probabilities(surname_series, zip_series)
 
-    sg_json = pd.DataFrame.to_json(sg_results)
-    return(sg_json)
-
-    # END OF SURGEO API; END OF TO-DO @KAYLA
+    return run_surgeo(surname=surname, 
```

Comment:
nitty gritty thing — `run_surgeo` vs. just `zrp`? either take out or add `run_` prefix

Reply:
Will do. run_surgeo was because "surgeo" is a protected keyword (the name of the actual surgeo module)